### PR TITLE
Update SWBD DA configuration

### PR DESCRIPTION
**Pull request by akreal**
- Extend `maxlenratio` to specify a constant output length limit and set it to 1 for SWBD DA.
- Use Conformer, SpecAug, speed perturbation for SWBD DA.

### CODE REVIEW

File: egs2/swbd_da/asr1/conf/tuning/train_asr_conformer.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+batch_type: numel
+batch_bins: 7000000
+accum_grad: 1
+max_epoch: 35
```
Comment (siddhu001) [inline egs2/swbd_da/asr1/conf/tuning/train_asr_conformer.yaml:4]: Can you try increasing epochs? Maybe adding train/val loss curve will give more intuition if increasing number of epochs will help.

File: espnet/nets/beam_search.py [hunk 2]
```diff
@@ -352,6 +354,8 @@ def forward(
         # set length bounds
         if maxlenratio == 0:
             maxlen = x.shape[0]
+        elif maxlenratio < 0:
```
Comment (ftshijt) [inline espnet/nets/beam_search.py:357]: Could you also consider adding a unit test for this modification?

Comment (akreal) [reply espnet/nets/beam_search.py:357]: Yes, sure. Is it fine now, or it should be done another way?

Comment (ftshijt) [reply espnet/nets/beam_search.py:357]: LGTM! Thank you.

**Comment by sw005320** [comment]
I think it would be good to have more documentation about the fixed-length output option.
https://github.com/espnet/espnet/blob/1f04ac9d6a12321f18e4ef61f840dff368425a70/doc/tutorial.md#ctc-attention-and-hybrid-ctcattention would not be the best place, but at least it discusses `maxlenratio` and please add this usage here.

Ideally, we want to split the descriptions about training and inference in https://github.com/espnet/espnet/blob/1f04ac9d6a12321f18e4ef61f840dff368425a70/doc/tutorial.md#ctc-attention-and-hybrid-ctcattention and discuss the conventional beam search with maxlenratio etc., and the special case with the fixed-length output option for the classification task.

**Comment by akreal** [comment]
> I think it would be good to have more documentation about the fixed-length output option.
Alright, I'll take a look at it.
